ts/navigation/AppStackNavigator.tsx

```diff
@@ -86,7 +84,7 @@ const InnerNavigationContainer = (props: { children: React.ReactElement }) => {
 
   const linking: LinkingOptions = {
     enabled: !isTestEnv, // disable linking in test env
-    prefixes: [IO_INTERNAL_LINK_PREFIX],
+    prefixes: [IO_INTERNAL_LINK_PREFIX, "https://continua.io.pagopa.it"],
```

Comment:
What do you think about moving this in a config file? I personally don't appreciate to have it directly in the linking configuration of `react-navigation`

Reply:
just moved here [8847b5d](https://github.com/pagopa/io-app/pull/4568/commits/8847b5d56b85b23cb47b63452a7396e5eb45edfe) like internal prefix.